lib/screens/authentication/registration_screen.dart

```diff
@@ -124,3 +59,490 @@ class RegistrationBody extends StatelessWidget {
     );
   }
 }
+
+class StepperForm extends StatefulWidget {
+  @override
+  _StepperFormState createState() => _StepperFormState();
+}
+
+class _StepperFormState extends State<StepperForm> {
+  int activeStep = 1; // Initial step set to 5.
+
+  int upperBound = 3;
+  bool maximum = false;
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: Column(
+        children: [
+          NumberStepper(
+            numbers: [
+              1,
+              2,
+              3,
+            ],
+
+            // activeStep property set to activeStep variable defined above.
+            activeStep: activeStep,
+
+            // This ensures step-tapping updates the activeStep.
+            onStepReached: (index) {
+              setState(() {
+                activeStep = index;
+              });
+            },
+          ),
+          Expanded(
+            child: FittedBox(
+              child: UserForm(
+                activeStep: activeStep,
+              ),
+            ),
+          ),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              previousButton(),
+              nextButton(),
+            ],
+          ),
+        ],
+      ),
+    );
+  }
+
+  /// Returns the next button.
+  Widget nextButton() {
+    return MaterialButton(
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.circular(30),
+      ),
+      color: Colors.orange,
+      padding: EdgeInsets.all(12),

```

Comment:
@wansom Add a const here and also to all padding or margin properties for performance optimization

Reply:
done